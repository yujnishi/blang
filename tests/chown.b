main() {
    printf("return code is %d*n",chown("/hoge",0));
    return(0);
}
