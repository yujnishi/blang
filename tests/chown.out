return code is -1
